Lib/test/test_os.py

```diff
@@ -1041,18 +1051,26 @@ def test_or_operator(self):
         new_vars_items = new_vars_dict.items()
         self.assertIs(NotImplemented, os.environ.__or__(new_vars_items))
 
+        self._test_underlying_process_env('_A_', '')
+        self._test_underlying_process_env('_B_', '')
```

Comment:
Do we gain anything by testing for `_B_`? It seems it's handled exactly the same as `_A_`. (I'm asking because this test runs a subprocess, which is pretty expensive compared to almost everything else these tests do.)

Reply:
Another option could be checking all three with one subprocess.

Reply:
But that would uglify the code. What's gained by checking three instead of two?

Reply:
I don't think anything is gained other than the code "feeling" complete in what it's testing. Admittedly, it's not actually testing different behavior.

The additional subprocess calls aren't very expensive though - it's only adding between 10 & 20 milliseconds. That being said, I don't have a strong feeling about it, so I'd be willing to remove the `_B_` tests.

Reply:
It's still something, and not everybody's machine is as fast. Let's cut the `_B_` tests -- it's too high a price for a sense of completeness or symmetry.